### ISSUE TITLE
Set Default Aspect Ratio/Crop for both Fullscreen and PiP

### ISSUE THREAD

**Issue by StarlightDesign** [lhc70000/iina#553]
**System and IINA version:**

- macOS 10.12.4
- IINA 0.0.7

**Expected behavior:**
Select a default crop/aspect ratio in the preferences and remember the setting for every video.

**Actual behavior:**
No such setting exists, have to open settings each time and adjust the crop/aspect ratio.

Requesting this because most MacBooks nowadays are running on the 16:10 aspect ratio, requiring the 16:10 crop to be turned on to avoid black bars along the top of most 16:9 videos. Cropping to 16:10 has the adverse effect of showing black bars on the left and right sides  for the Picture-in-Picture window. If there's a way to offset this by setting a PiP default crop/aspect ratio as well, that would be the best outcome.

<img width="1440" alt="16-by-9 ratio on 2016 macbook pro" src="https://cloud.githubusercontent.com/assets/8271041/24603278/fa7bda1c-1814-11e7-917d-002278ecd36c.png">
<img width="343" alt="pip after 16-by-10 crop" src="https://cloud.githubusercontent.com/assets/8271041/24603279/faa5d7d6-1814-11e7-9d1d-0deb0c8f9d3e.png">

**Comment by perara-chen** [comment]
I also think this is a must have enhancement. I also have the same issue on my new Mac Air M1.

**Comment by YvesMaud** [comment]
Please!

**Comment by lhaeger** [comment]
Yes, yes, yes! So tiring to start every video with that Open Settings, Crop to 16:10, Close Settings dance...

**Comment by lhaeger** [comment]
Seems as if setting panscan=1.0 in extended mvp options fixes this. I get the original aspect ratio in regular windows and 16:10 in full screen on my MacBook Pro as a result. Can anyone try and confirm this?

**Comment by YvesMaud** [comment]
Yeah, it worked for me, but then i decided to redo it, and now it shows me this… :(
<img width="260" alt="Screenshot 2023-01-19 at 20 02 20" src="https://user-images.githubusercontent.com/114612350/213511713-31e82550-c722-49a3-ba68-9d172ced679a.png">
